session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -913,6 +914,27 @@ public SessionDataSet executeQueryStatement(String sql)
         execResp.isIgnoreTimeStamp());
   }
 
+  public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, long endTime)
+          throws StatementExecutionException, IoTDBConnectionException {
+
+    String statement = "raw data query";
```

Comment:
please make the statement is a constant field to avoid too many local variables to created and GC.